### PR TITLE
VPE TF Chnges

### PR DESCRIPTION
**Pull request by nimisha0092**

### CODE REVIEW

File: ibm/resource_ibm_is_virtual_endpoint_gateway.go [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Endpoint gateway health state",
+			},
+			isVirtualEndpointGatewayLifecycleState: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Endpoint gateway lifecycle state",
+			},
+			isVirtualEndpointGatewayIPs: {
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway.go:82]: From API docs isVirtualEndpointGatewayIPs is a required attribute

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	isVirtualEndpointGatewayID                   = "gateway_id"
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go:13]: use gateway instead  of gateway_id to remain in sync with all VPC reosurces

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+	isVirtualEndpointGatewayIPID                 = "reserved_ip_id"
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go:14]: here also reserved_ip_id change to reserved_ip

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+	parts, err := idParts(d.Id())
+	if err != nil {
+		return err
+	}
+	gatewayID := parts[0]
+	ipID := parts[1]
+	opt := sess.NewRemoveEndpointGatewayIPOptions(gatewayID, ipID)
+	response, err := sess.RemoveEndpointGatewayIP(opt)
+	if err != nil {
+		log.Printf("Remove Endpoint Gateway IP failed: %v", response)
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go:165]: Check for 404 condition also and set d.SetID("")

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string,ForceNew) Endpoint gateway name
+* `target` - (Required, stringList) The unique identifier of VPN gateway(ID).
+  * `name` - (Required, string) The IP address of the peer VPN gateway.
+  * `resource_type`- (Required, string) The preshared key.
+* `vpc` - (Required, string) List of CIDRs for this resource,optional for mode route.
+* `ips` -  (Optional, stringList)Endpoint gateway resource group
```
Comment (hkantare) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:68]: its not the reosurce group
Endpoint gateway resource group

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string,ForceNew) Endpoint gateway name
+* `target` - (Required, stringList) The unique identifier of VPN gateway(ID).
```
Comment (hkantare) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:64]: The unique identifier of VPN gateway(ID). this defination is wrong

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown (hunk 5 above)
Comment (hkantare) [reply website/docs/r/is_virtual_endpoint_gateway.html.markdown:68]: check all description for all fields they are pointing to wrong description please use API docs/cli comamnds description

File: website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the VPN gateway connection. The id is composed of \<gateway_id\>/\<gateway_ip_id\>.
+* `name` - Endpoint gateway IP name
+* `created_at` -  Endpoint gateway IP created date and time
+* `resource_type` -  Endpoint gateway IP resource type
```
Comment (hkantare) [inline website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown:42]: It gives "address" also one of the computed

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+data "ibm_is_virtual_endpoint_gateway" "data_test" {    
+    name = ibm_is_virtual_endpoint_gateway.endpoint_gateway.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- name-(Required,string)Endpoint gateway name
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway.html.markdown:25]: check other docs format for this line and update it

File: website/docs/d/is_virtual_endpoint_gateways.html.markdown [hunk 9]
```diff
@@ -0,0 +1,45 @@
+---
+layout: "ibm"
+page_title: "IBM : docs-ibm-datasource-is-virtual-endpoint-gateways"
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateways.html.markdown:3]: change here too..

File: website/docs/d/is_virtual_endpoint_gateways.html.markdown [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+sidebar_current: "docs-ibm-virtual_endpoint_gateways"
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateways.html.markdown:4]: change here too..

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 11]
```diff
@@ -0,0 +1,101 @@
+---
+layout: "ibm"
+page_title: "IBM : docs-ibm-resource-is-virtual-endpoint-gateway"
```
Comment (KumarGanesanIBM) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:3]: check other resource document and follow the same

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+sidebar_current: "ibm_is_virtual_endpoint_gateway"
```
Comment (KumarGanesanIBM) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:4]: check other resource document and follow the same

File: website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown [hunk 13]
```diff
@@ -0,0 +1,56 @@
+---
+layout: "ibm"
+page_title: "IBM : docs-ibm-resource-is-virtual-endpoint-gateway-ip"
```
Comment (KumarGanesanIBM) [inline website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown:3]: check other resource document and follow the same

File: website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+sidebar_current: "ibm_is_virtual_endpoint_gateway_ip"
```
Comment (KumarGanesanIBM) [inline website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown:4]: check other resource document and follow the same

File: website/ibm.erb [hunk 15]
```diff
@@ -461,6 +461,15 @@
             <li<%= sidebar_current("docs-ibm-datasource-is-vpc") %>>
               <a href="/docs/providers/ibm/d/is_vpc.html">is_vpc</a>
             </li>
+            <li<%= sidebar_current("docs-ibm-datasource-is-virtual-endpoint-gateway") %>>
```
Comment (KumarGanesanIBM) [inline website/ibm.erb:464]: make sure all the sidebar content is matched with the docs

File: website/ibm.erb [hunk 16]
```diff
@@ -1059,6 +1068,12 @@
             <li<%= sidebar_current("docs-ibm-resource-is-vpc") %>>
               <a href="/docs/providers/ibm/r/is_vpc.html">is_vpc</a>
             </li>
+            <li<%= sidebar_current("docs-ibm-resource-is-virtual-endpoint-gateway") %>>
```
Comment (KumarGanesanIBM) [inline website/ibm.erb:1071]: make sure all the sidebar content is matched with the docs

**Review by KumarGanesanIBM — CHANGES_REQUESTED** [review]

File: website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+The following arguments are supported:
+
+- gateway_id(Required,string)- Endpoint gateway ID
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+- `ips` - Endpoint gateway id
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown:32]: change the description for this attribute to `Collection of endpoint gateway reserved IPs`

File: website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+data "ibm_is_virtual_endpoint_gateway_ips" "data_test1" {
+  gateway_id     = ibm_is_virtual_endpoint_gateway.endpoint_gateway.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- gateway_id(Required,string)- Endpoint gateway ID
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown:25]: check other docs and change this line accordingly.

File: website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+- `ips` - Endpoint gateway id
+- `reserved_ip` - Endpoint gateway IP id
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown:33]: change this description to `Endpoint gateway reserved IP id`

File: website/docs/d/is_virtual_endpoint_gateways.html.markdown [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+- `id` - Endpoint gateway id
+- `name` -  Endpoint gateway name
+- `resource_group` - Endpoint gateway resource type
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateways.html.markdown:34]: change this description to `Endpoint gateway resource group id`

**Review by KumarGanesanIBM — CHANGES_REQUESTED** [review]

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+The following arguments are supported:
+
+- name(Required,string)- Endpoint gateway name
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+- `resource_group` - Endpoint gateway resource type
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway.html.markdown:32]: Please give correct description for each attribute. also check other attribute has the correct description.

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+- `created_at` - Endpoint gateway created date and time
+- `health_state` - Endpoint gateway health state
+- `lifecycle_state` - Endpoint gateway lifecycle state
+- `ips` -  Endpoint gateway resource group
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway.html.markdown:36]: Please give correct description for each attribute. also check other attribute has the correct description.

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+data "ibm_is_virtual_endpoint_gateway" "data_test" {    
+    name = ibm_is_virtual_endpoint_gateway.endpoint_gateway.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- name(Required,string)- Endpoint gateway name
```
Comment (KumarGanesanIBM) [inline website/docs/d/is_virtual_endpoint_gateway.html.markdown:25]: format is wrong. check the other attribute and change it.

**Review by KumarGanesanIBM — CHANGES_REQUESTED** [review]

File: ibm/resource_ibm_is_virtual_endpoint_gateway.go (hunk 1 above)
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway.go:81]: I can see isVirtualEndpointGatewayIPs(ips) as an optional attribute.
https://pages.github.ibm.com/riaas/api-spec/spec_aspirational/#/Endpoint%20gateways/create_endpoint_gateway

File: ibm/data_source_ibm_is_virtual_endpoint_gateway.go [hunk 24]
```diff
@@ -0,0 +1,152 @@
...
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceIBMISEndpointGateway() *schema.Resource {
+	return &schema.Resource{
+		Read:     dataSourceIBMISEndpointGatewayRead,
+		Importer: &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/data_source_ibm_is_virtual_endpoint_gateway.go]: handled

File: ibm/data_source_ibm_is_virtual_endpoint_gateway.go [hunk 25]
```diff
@@ -0,0 +1,152 @@
...
+			d.Set(isVirtualEndpointGatewayIPs, flattenIPs(result.Ips))
+			d.Set(isVirtualEndpointGatewayResourceGroupID, result.ResourceGroup.ID)
+			d.Set(isVirtualEndpointGatewayTarget, flattenEndpointGatewayTarget(
+				result.Target.(*vpcv1.EndpointGatewayTarget)))
+			d.Set(isVirtualEndpointGatewayVpcID, result.VPC.ID)
+			found = true
+			break
+		}
+	}
+	if found == false {
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/data_source_ibm_is_virtual_endpoint_gateway.go]: handled

File: ibm/data_source_ibm_is_virtual_endpoint_gateway.go [hunk 26]
```diff
@@ -0,0 +1,152 @@
...
+		return fmt.Errorf("Given virtual endpoint gateway does not exist. " +
+			"Please specify correct name")
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/data_source_ibm_is_virtual_endpoint_gateway.go]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway.go [hunk 27]
```diff
@@ -0,0 +1,333 @@
...
+							Computed:    true,
+							Description: "The IPs id",
+						},
+						isVirtualEndpointGatewayIPsName: {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "The IPs name",
+						},
+						isVirtualEndpointGatewayIPsSubnetID: {
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway.go]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go (hunk 2 above)
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go:13]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go (hunk 3 above)
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go (hunk 4 above)
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go:165]: handled

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown (hunk 5 above)
Comment (nimisha0092) [reply website/docs/r/is_virtual_endpoint_gateway.html.markdown]: handled

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown (hunk 6 above)
Comment (nimisha0092) [reply website/docs/r/is_virtual_endpoint_gateway.html.markdown]: handled

File: website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown (hunk 7 above)
Comment (nimisha0092) [reply website/docs/r/is_virtual_endpoint_gateway_ip.html.markdown:42]: handled

File: ibm/data_source_ibm_is_virtual_endpoint_gateways_test.go [hunk 28]
```diff
@@ -0,0 +1,35 @@
...
+
+func TestAccIBMISVirtualEndpointGatewaysDataSource_basic(t *testing.T) {
+	resName := "data.ibm_is_virtual_endpoint_gateways.test1"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				ExpectNonEmptyPlan: true,
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/data_source_ibm_is_virtual_endpoint_gateways_test.go]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway.go [hunk 29]
```diff
@@ -0,0 +1,333 @@
...
+	d.Set(isVirtualEndpointGatewayVpcID, result.VPC.ID)
+	return nil
+}
+
+func resourceIBMisVirtualEndpointGatewayDelete(d *schema.ResourceData, meta interface{}) error {
+	sess, err := vpcClient(meta)
+	if err != nil {
+		return err
+	}
+	// read option
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway.go]: handled

File: website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown (hunk 17 above)
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown]: handled

File: website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown (hunk 18 above)
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown:25]: handled

File: website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown (hunk 19 above)
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown:33]: handled

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 30]
```diff
@@ -0,0 +1,93 @@
...
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string,ForceNew) Endpoint gateway name
+* `target` - (Required, stringList) Endpoint gateway target
+  * `name` - (Required, string) The target name
+  * `resource_type`- (Required, string) The target resource type
+* `vpc` - (Required, string) The VPC id
+* `ips` -  (Required, stringList)Endpoint gateway resource group
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply website/docs/r/is_virtual_endpoint_gateway.html.markdown]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway_test.go [hunk 31]
```diff
@@ -0,0 +1,224 @@
...
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccIBMISVirtualEndpointGateway_Basic(t *testing.T) {
+	var endpointGateway string
+	name := "ibm_is_virtual_endpoint_gateway.endpoint_gateway"
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		// No requirement for CheckDestory of this resource as by reaching this point it must have already been deleted from CIS.
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway_test.go]: handled

File: website/docs/d/is_virtual_endpoint_gateways.html.markdown [hunk 32]
```diff
@@ -0,0 +1,45 @@
...
+- `created_at` - Endpoint gateway created date and time
+- `health_state` - Endpoint gateway health state
+- `lifecycle_state` - Endpoint gateway lifecycle state
+- `ips` -  Endpoint gateway resource group
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateways.html.markdown]: handled

File: website/docs/d/is_virtual_endpoint_gateways.html.markdown [hunk 33]
```diff
@@ -0,0 +1,45 @@
...
+    - `id` - The IPs id
+    - `name` - The IPs name
+    - `resource_type` - Endpoint gateway IP resource type
+- `target` -  Endpoint gateway target
```
[the remark this answers is not in the document]
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateways.html.markdown]: handled

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown (hunk 21 above)
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateway.html.markdown]: handled

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown (hunk 22 above)
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateway.html.markdown]: handled

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown (hunk 23 above)
Comment (nimisha0092) [reply website/docs/d/is_virtual_endpoint_gateway.html.markdown]: handled

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 34]
```diff
@@ -0,0 +1,93 @@
...
+  * `name` -  (Optional, string)Endpoint gateway resource group IPs name
+  * `subnet` -  (Optional, string)Endpoint gateway resource group Subnet id
+  * `resource_type` -  (Computed, string)Endpoint gateway resource group VPC Resource Type
+* `resource_group` -  (Optional, string,ForceNew)The resource group id  
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the VPN gateway connection. The id is composed of \<virtual_endpoint_gateway_id>
```
Comment (hkantare) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:79]: its just the id of the gateway

File: ibm/resource_ibm_is_virtual_endpoint_gateway_test.go [hunk 35]
```diff
@@ -0,0 +1,223 @@
...
+		resource_group = data.ibm_resource_group.test_acc.id
+	}
+	resource "ibm_is_subnet" "testacc_subnet" {
+		name = "test-vpe-subnet1"
+		vpc = ibm_is_vpc.testacc_vpc.id
+		zone = "%[1]s"
+		ipv4_cidr_block = "%[2]s"
+		resource_group = data.ibm_resource_group.test_acc.id
+	}
+	resource "ibm_is_virtual_endpoint_gateway" "endpoint_gateway" {
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway_test.go:183]: Don't hardcode the names of the resouces test-vpe-subnet1,test-vpe-network-vpc...like this in all testcases
See the remaining VPC testcases and use the same

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go [hunk 36]
```diff
@@ -0,0 +1,214 @@
...
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	isVirtualEndpointGatewayID                   = "gateway_id"
```
Comment (hkantare) [inline ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go:13]: change gateway_id to gateway

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown (hunk 34 above)
Comment (nimisha0092) [reply website/docs/r/is_virtual_endpoint_gateway.html.markdown]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway_test.go (hunk 35 above)
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway_test.go:196]: handled

File: ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go (hunk 36 above)
Comment (nimisha0092) [reply ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go]: handled

**Pull request merged**